data/update/mysql/dbupdate-6.11.0-to-6.12.0.sql

```diff
@@ -0,0 +1,5 @@
+--
```

Comment:
btw missing version bump

Reply:
I made similar additions that were in 6.10.0-to-6.11.0.sql (x2), please verify these were as expected.